.github/workflows/ci.yml

```diff
@@ -23,7 +22,7 @@ jobs:
         with:
           cache: true
       - name: Lint
-        run: flutter format --output=none --set-exit-if-changed packages/flutter
+        run: flutter format --output=none --set-exit-if-changed /
```

Comment:
since now we are using dart not Flutter, any Flutter-related job is not needed anymore.